improves section about routes (#7675)
withastro/docs: src/content/docs/en/guides/integrations-guide/cloudflare.mdx

### Routes

> [Cloudflare routing](https://developers.cloudflare.com/pages/platform/functions/routing/#functions-invocation-routes) uses a `_routes.json` file to determine which requests are routed to the SSR function and which are served as static assets. By default, a `_routes.json` file will be automatically generated based on your project. You can also create a custom `_routes.json` file to override the default behavior.  You can [specify additional patterns to follow](#routesextend) in your adapter config, or create your own custom `_routes.json` file to fully override the automatic generation.

Reviewer: ```suggestion
[Cloudflare routing](https://developers.cloudflare.com/pages/platform/functions/routing/#functions-invocation-routes) uses a `_routes.json` file to determine which requests are routed to the SSR function and which are served as static assets. By default, a `_routes.json` file will be automatically generated for your project based on its files and configuration.

You can [specify additional routing patterns to follow](#routesextend) in your adapter config, or create your own custom `_routes.json` file to fully override the automatic generation.
```
